1001-2000/LIT1460Gadlah.xml

```diff
@@ -57,45 +57,163 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: text record</change>
          <change who="MV" when="2017-11-13">Additions</change>
+         <change who="MV" when="2020-05-12">Added texparts, relation and bibl</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <div type="bibliography">
+            <listRelation>
+               <relation name="ecrm:P129i_is_subject_of" active="PRS1948Anne" passive="LIT1460Gadlah"></relation>      
+            </listRelation>
             <listBibl type="editions">
                <bibl>
                   <ptr target="bm:Budge1900Miracles"/>
-                  <citedRange>84-107</citedRange>
+                  <citedRange>84-104</citedRange>
                </bibl>
             </listBibl>
-            <listBibl type="translation">
+            <listBibl type="translation" xml:lang="en">
                <bibl>
                   <ptr target="bm:Budge1900Miracles"/>
+                  <citedRange>159-207</citedRange>
+               </bibl>
+               <bibl>
+                  <ptr target="bm:Budge1922Mary"/>
                </bibl>
             </listBibl>
          </div>
+         
          <div type="edition">
             <div type="textpart" subtype="chapter" n="1" xml:id="Prologue">
                <label>Prologue</label>
                <div type="textpart" subtype="incipit" xml:lang="gez">
```

Comment:
could you add a little `<note>` in the edition stating from where these texts are copied (I assume Budge, but better to be a bit too explicit) here and in the following records?